tests/run/cimport_constants.srctree

```diff
@@ -0,0 +1,70 @@
+PYTHON setup.py build_ext --inplace
+PYTHON -c "import mymodule"
+PYTHON testerr.py
+
+######## setup.py ########
+
+from Cython.Build.Dependencies import cythonize
+from distutils.core import setup
+
+setup(
+  ext_modules = cythonize(["constants.pyx", "mymodule.pyx"]),
+)
+
+######## constants.pxd ########
+
+cdef const int my_c_pyx
+cdef const int my_c_pxd = 6
+
+######## constants.pyx ########
+
+cdef const int my_c_pyx = 5
+
+######## mymodule.pyx ########
+
+cimport constants
+cdef int pyx_c = constants.my_c_pyx
+cdef int pxd_c = constants.my_c_pxd
+assert pyx_c == 5
+assert pxd_c == 6
+
+######## errmodule.pyx ########
+
+cimport constants
+cdef int[constants.my_c_pyx] my_arr
+cdef int[constants.my_c_pxd] my_arr
+
+######## testerr.py ########
+import os
+import sys
+from contextlib import contextmanager
+from Cython.Compiler.Errors import CompileError
+from Cython.Build.Dependencies import cythonize
+
+if sys.version_info[0] < 3:
+    from StringIO import StringIO
+else:
+    from io import StringIO
+
+@contextmanager
+def assert_stderr(expected_errors):
```

Comment:
This test is based on following code: https://github.com/cython/cython/blob/1078cc635c164b594f4eff8c2fdc401412b8f0e7/tests/run/relative_import_leak.srctree#L16C3-L27

We should build some common infrastructure for it in some near future I suppose.